.github/workflows/rust.yml

```diff
@@ -55,7 +55,17 @@ jobs:
         if: always()
         with:
           command: clippy
-          args: --workspace --all-targets -- -D warnings
+          args: --all-targets -- -D warnings
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+          components: clippy
+      - name: lint fuzz
+        run: |
+          cd fuzz
+          cargo clippy -- -D warnings
```

Comment:
Do we need something like this for formatting also?

Reply:
To my surprise no, as it seems that the `--all` option will apply formatting to all packages regardless of their membership in the root workspace `workspace.default-members` list. 

To demonstrate, I've added some format errors into the fuzz package and ran the action on a branch of my fork. Happily it is caught [as expected by the lint check](https://github.com/jafow/quinn/runs/1460865241?check_suite_focus=true).